kubernetes/groupversion_info.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2022.
```

Comment:
Do you mind fixing this license header? Maybe we can use the `addheaders` script from other projects.

Reply:
sure, fixed.